pkcs11/pkcs11_rsa.c

```diff
@@ -321,6 +321,83 @@ static int rsa_sign_verify_pss(int devId)
 }
 #endif /* ifdef WC_RSA_PSS */
 #endif /* ifndef NO_RSA */
+static int rsa_encrypt_decrypt(int devId)
+{
+    int    ret = 0;
+    byte   plain[128], out[2048/8], dec[2048/8];
+    word32 plainSz, outSz, decSz;
+    RsaKey pub;
+    RsaKey priv;
+
+    memset(plain, 9, sizeof(plain));
```

Comment:
Why `9`? Please add inline code comment... even if its just saying initialize the input with 9's.